R/sparsevctrs.R

```diff
@@ -1,3 +1,22 @@
+#' Using sparse data with recipes
+#' 
+#' [recipe()], [prep()], and [bake()] all accept sparse tibbles from the 
+#' `sparsevctrs` package and sparse matrices from the `Matrix` package. Sparse
+#' matrices are converted to sparse tibbles internally as each step expects a 
+#' tibble as its input, and is expected to return a tibble as well.
+#' 
+#' Several steps work with sparse data. A step can either work with  sparse 
+#' data, ruin sparsity, or create sparsity. The documentation for each  step 
+#' will indicate whether it will work with sparse data or create sparse columns. 
+#' If nothing is listed it is assumed to ruin sparsity.
```

Comment:
Been sitting with these lines for a moment and not sure what I'd concretely suggest, but "ruin" doesn't feel like the word we want here (or its usage just needs clarification. Maybe we could link to some sparsevctrs docs here to clarify materializing sparse data?

Reply:
this section might be better once In have implemented it in a few steps and is able to make references